action.yaml

```diff
@@ -37,6 +37,11 @@ inputs:
     description: 'Set to "true" to Destroy the stack'
   aws_resource_identifier:
     description: 'Set to override the AWS resource identifier for the deployment.  Defaults to `${org}-{repo}-{branch}`.  Use with destroy to destroy specific resources.'
+  domain_name:
+    description: 'Define the root domain name for the application. e.g. app.com'
+    required: true
+  sub_domain:
+    description: 'Define the sub-domain part of the URL. Defaults to `${org}-${repo}-{branch}`'
```

Comment:
I think these will need to be converted to environment variables here:
https://github.com/bitovi/github-actions-node-app-to-aws-vm/pull/9/files#diff-fab4d7fb461bc6fbe9587f6c03fff98102b1c744145edcf2a993f2ff7cb05a0dR59

Reply:
@LeoDiazL do github action inputs (such as `domain_name`) automatically get converted into environment variables that the scripts in the runner can use (such as `DOMAIN_NAME`)?